brainframe/api/stubs/zone_statuses.py

```diff
@@ -51,6 +52,7 @@ def zone_status_iterator():
             resp = self._get(req, timeout=timeout)
 
             packets = resp.iter_lines(delimiter=b"\r\n")
+            timeout_start = time.time()
```

Comment:
Shouldn't the timeout reset every time a zone status is successfully sent to the client? Otherwise I would assume this API times out after the timeout has elapsed even during normal use.

Reply:
Also, this variable is never used?

Reply:
This line has been removed from bb9725675ae31c60881a894ad5cd5003ac277cad